packages/block-library/src/image/editor.scss

```diff
@@ -124,3 +124,13 @@
 		caption-side: bottom;
 	}
 }
+
+.block-editor-block-list__block[data-type="core/image"] {
```

Comment:
Probably simpler to target `.wp-block-image` instead of using an attribute selector (`[data-type="core/image"]`).

Reply:
Yes, simpler but not specific enough as I need to override how it flex direction is set by default, and the default selector for this is using the data attribute selector.